src/lib.rs

```diff
@@ -1351,6 +1351,8 @@ impl Build {
             },
         );
 
+        cmd.arg("-c");
```

Comment:
Nit: Please add a comment linking to previous issues.
```suggestion
        // Checking for compiler flags does not require linking (and we _must_
        // avoid making it do so, since it breaks cross-compilation when the C
        // compiler isn't configured to be able to link).
        // https://github.com/rust-lang/cc-rs/issues/1423
        cmd.arg("-c");
```